web/src/beta/lib/core/engines/Cesium/Feature/index.tsx

```diff
@@ -161,9 +167,23 @@ export default function Feature({
           const [C] = components[k] ?? [];
           const isVisible = layer.layer.visible !== false && !isHidden;
 
+          const useSceneSphericalHarmonicCoefficients =
+            !!props.sceneProperty?.light?.sphericalHarmonicCoefficients;
+          const useSceneSpecularEnvironmentMaps =
+            !!props.sceneProperty?.light?.specularEnvironmentMaps;
+
+          const use3dtilesIBL =
+            layer?.layer?.type === "simple" &&
+            (!!layer?.layer?.["3dtiles"]?.sphericalHarmonicCoefficients ||
+              !!layer?.layer?.["3dtiles"]?.specularEnvironmentMaps);
```

Comment:
```suggestion
          const use3dtilesSphericalHarmonicCoefficients =
            layer?.layer?.type === "simple" && !!layer?.layer?.["3dtiles"]?.sphericalHarmonicCoefficients;
         const use3dtilesSpecularEnvironmentMaps =
            layer?.layer?.type === "simple" && !!layer?.layer?.["3dtiles"]?.specularEnvironmentMaps;
```
I think we should separate the key to make sure to update the component.

Reply:
For example, `3d tiles IBL` seems not to be working.

Reply:
Updated. 🙏🏻